config/clusters/openscapes/staging.values.yaml

```diff
@@ -1,8 +1,13 @@
 basehub:
+  dask-gateway:
+    enabled: true
   userServiceAccount:
     annotations:
       eks.amazonaws.com/role-arn: arn:aws:iam::783616723547:role/openscapeshub-staging
   jupyterhub:
+    custom:
+      daskGateway:
+        enabled: true
```

Comment:
Please feel free to dismiss this comment as I don't feel very strongly about it, but I cannot seem to decide if having these two be named the similar (`dask-gateway` and `daskGateway`) is confusing or it's good because it "ties" them together 😄 

I think personally I'd name `jupyterhub.custom.daskGateway` `jupyterhub.custom.daskhub` `jupyterhub.custom.daskhubType` for an easier transition from what we have. But then again I'm bad with names so please feel free to ignore.

Also, I think it's worth adding them to the schema definition for reference although not mandatory.

Reply:
Ideally I'd love for you to pick up this and finish it, and you should name them as you see fit!

Reply:
I think it would be useful for it to be consistent with everything else under `jupyterhub.custom`, whatever that is.